pkg/scalers/mysql_scaler.go

```diff
@@ -104,14 +164,35 @@ func newMySQLConnection(meta *mySQLMetadata, logger logr.Logger) (*sql.DB, error
 		logger.Error(err, fmt.Sprintf("Found error when opening connection: %s", err))
 		return nil, err
 	}
+
 	err = db.Ping()
 	if err != nil {
 		logger.Error(err, fmt.Sprintf("Found error when pinging database: %s", err))
 		return nil, err
 	}
+
+	setConnectionPoolConfiguration(meta, db)
+
 	return db, nil
 }
 
+// setConnectionPoolConfiguration configures the MySQL connection pool settings
+// based on the parameters provided in mySQLMetadata. If a setting is zero, it
+// is left at its default value.
+func setConnectionPoolConfiguration(meta *mySQLMetadata, db *sql.DB) {
```

Comment:
I think that maybe we could hide this parameter for the moment and share them if there is any request from users. As they can have different values, this could be so difficult to debug.
@kedacore/keda-core-contributors ?

Reply:
Hey, yeah, just to clarify, did you mean to remove all the new parameters, thus making the globally shared connection pools the default, or just the connection pool tuning parameters? 

I also thought a MySQL scaler v2 might fit as well for this, if that's of interest at all?

Reply:
I'm not sure if adding a v2 scaler makes sense as we could potentially implement a connection pool as part of https://github.com/kedacore/keda/issues/6360.

Definitivelly, supporting this in mysql/postgre scalers is worth, but adding v2 scaler to include this feature could be an overkill as it won't make sense if we address a global connection pool

Reply:
@kedacore/keda-core-contributors ?

Reply:
ping @kedacore/keda-core-contributors ?

Reply:
Does making it an experimental instead of hiding parameters make it a better choice, since it helps to collect user opinions? And, I think there is no need to create a v2 scaler as the basic logic of the scaler has not changed.